spec/unit/readerfooter_spec.lua

```diff
@@ -305,20 +309,25 @@ describe("Readerfooter module", function()
         local footer = readerui.view.footer
         local horizontal_margin = Screen:scaleBySize(10)*2
         footer:onUpdateFooter()
-        assert.is.same(370, footer.text_width)
+        -- Account for trimming of the leading 0 in the AM
+        local expected = is_am() and 362 or 370
```

Comment:
Is it really `AM` vs. `PM` that causes the difference?
And not `1:12 AM` being shorter than `11:12 AM` (both AM) ?